project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.2.0")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.34")
```

Comment:
what is this for?

Reply:
It was used for the migration but I can get rid of it now.